src/ring_theory/polynomial/cyclotomic/basic.lean

```diff
@@ -853,4 +860,70 @@ end
 
 end expand
 
+section char_p
+
+/-- If `R` is of characteristic `p` and `¬p ∣ n`, then
+`cyclotomic (n * p) R = (cyclotomic n R) ^ (p - 1)`. -/
+lemma cyclotomic_mul_prime_not_dvd_eq_pow (R : Type*) {p n : ℕ} [hp : fact (nat.prime p)] [ring R]
```

Comment:
```suggestion
lemma cyclotomic_mul_prime_eq_pow_of_not_dvd (R : Type*) {p n : ℕ} [hp : fact (nat.prime p)] [ring R]
```
Alternative would be `cyclotomic_mul_prime_eq_pow_sub_one` to contrast it clearly from the next lemma.